schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals
+(
+    id integer NOT NULL,
```

Comment:
- [OPTIONAL] It will be nice to make the id column not null and auto-incremented like so: `id int NOT NULL IDENTITY(1,1)`